libs/oeth/history/src/components/APYContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Divider, Stack, Typography } from '@mui/material';
+import { valueFormat } from '@origin/shared/components';
+import { theme } from '@origin/shared/theme';
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useAccount } from 'wagmi';
+import { useHistoryTableQuery } from '../queries.generated';
+import { isAddressEqual } from 'viem';
+
+export function APYContainer() {
+  const { address, isConnected } = useAccount();
+  const { data } = useHistoryTableQuery(
+    { addressId: address?.toLowerCase(), offset: 0 },
+    {
+      enabled:
+        isConnected &&
+        isAddressEqual(address, address?.toLowerCase() as `0x${string}`),
```

Comment:
`isAddressEqual` should be case insensitive, you can omit the lowerCase. Also, there's a helper `HexAddress` type for `0x{string}`

Reply:
i removed completely this assertion